packages/tests/web-test-runner.config.js

```diff
@@ -188,24 +188,11 @@ export default {
       },
     }),
   ],
-  filterBrowserLogs: ({args}) => {
-    if (
-      mode === 'dev' &&
-      typeof args[0] === 'string' &&
-      args[0].includes('in dev mode')
-    ) {
-      if (!seenDevModeLogs.has(args[0])) {
-        seenDevModeLogs.add(args[0]);
```

Comment:
Oh sorry, please remove `seenDevModeLogs`, no longer referenced.

Reply:
Actually, what is the behavior now w.r.t. the dev logs?

Reply:
They do not display. When running the tests, there is still a console log saying that you're using development or production builds.